Core/scripts/dirac-configure.py

```diff
@@ -400,9 +400,13 @@ def setExtensions(optionValue):
     result = bdc.syncCAs()
     if result['OK']:
       result = bdc.syncCRLs()
-  except BaseException:
+    if not result['OK']:
+      DIRAC.gLogger.warn('Failed to sync CAs and CRLs: %s' % result['Message'])
+  except ImportError:
```

Comment:
I don't think this makes sense, while the error message says "import" the exception from #4734 is `OSError`, so an ImportError is rather unlikely to happen here.

Reply:
For some reason, the `BlunderDeliveryClient` import was also wrapped with a `try / except` construct with an "import problem" message, so I left this exception.. but I think, you right. I don't see why there could be an `ImportError` exception, I'll exclude it from `try / except`..